### PR TITLE
Documentation page for gfm

### PR DESCRIPTION
**Pull request by ValentinaHutter**

### CODE REVIEW

**Comment by m-mohr** [comment]
This still needs a link in the menu?!

File: usecases/gfm/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+The operational implementation the GFM product includes the following key
+elements:
+- Downloading of worldwide Sentinel-1 SAR acquisitions (Level-1 IW GRDH)
+- Pre-processing of the downloaded Sentinel-1 data to backscatter data (SIG0)
+- Operational application of three fully automated flood mapping algorithms.
+- An ensemble-based approach is then used to combine the three flood extent
+  outputs of the individual flood algorithms.
+- Generation of the required GFM output layers, including Observed flood extent,
+  Reference water mask, Exclusion Mask and Likelihood Values
+- Web service-based access and dissemination of the GFM product output layers
```
Comment (SerRichard) [inline usecases/gfm/index.md:17]:
```suggestion
- Web service-based access and dissemination of the GFM product output layers.
```

File: usecases/gfm/index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Output layers used in openEO
+- Observed flood extent (ENSEMBLE of all three individual flood outputs)
+- Reference water mask (permament and seasonal water bodies)
+
+## Links
+https://extwiki.eodc.eu/GFM
+
+## Compute the maximum flood extent
+In this example, we have a closer look at an area in Pakistan, which was revaged by the unprecedented floods of 2022. The flooding caused severe damage and economic losses and is referred to as the worst flood in the history of Pakistan.
```
Comment (SerRichard) [inline usecases/gfm/index.md:27]:
```suggestion
In this example, we have a closer look at an area in Pakistan, which was ravaged by the unprecedented floods of 2022. The flooding caused severe damage and economic losses and is referred to as the worst flood in the history of Pakistan. 
```

File: usecases/gfm/index.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+The GHSL is available in `wgs84`. Therefore, the `tile_grid` for the GFM data was set to `wgs84` as well.
+
+<figure>
+    <img src="./gfm-flood-extent.png" alt="Flood extent">
+    <figcaption>Figure 1: Estimate of how the built-up surface was effected by the flood in Pakistan in September 2022. Some of the highest values of the GHSL can be found around 25.16 N 69.11 E, which marks Digri Tehsil, the second largest town of Mirpurkhas District, Pakistan. </figcaption>
+</figure>
+
+## Statistical analysis
+
+In the example given above, we picked the `sum` in `gfm_sum = gfm_data.reduce_dimension(dimension="time", reducer=sum)`. openEO provides a range of reducers to choose from. E.g.:
```
Comment (SerRichard) [inline usecases/gfm/index.md:82]:
```suggestion
In the example given above, we picked the `sum` in `gfm_sum = gfm_data.reduce_dimension(dimension="time", reducer=sum)`. OpenEO provides a range of reducers to choose from. E.g.:
```

File: usecases/gfm/index.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+# Save the result in Equi7Grid and as GeoTiff
+observed_water_tif = observed_water.save_result(format="GTiff", options={"tile_grid": "equi7"})
+```
+</template>
+</CodeSwitcher>
+
+## Explore the observed water
+
+The original GFM data is stored on the Equi7 Grid and the Asian Equi7 coordinate reference system. With the `tile_grid` parameter, the user can either keep the data like this, or pick a different CRS.
```
Comment (SerRichard) [inline usecases/gfm/index.md:130]:
```suggestion
The original GFM data is stored in the Equi7 Grid and the Asian Equi7 coordinate reference system. With the `tile_grid` parameter, the user can either keep the data like this, or pick a different CRS.
```

**Review by SerRichard — APPROVED** [review]
Some minor suggestions :smile:

**Pull request merged**